src/rosdep2/platforms/alpine.py

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env python
```

Comment:
shebang is not needed here (https://github.com/ros-infrastructure/rosdep/pull/630)